.all-contributorsrc

```diff
@@ -86,6 +86,15 @@
         "doc"
       ]
     },
+    {
+      "login": "sudhirtibrewal",
+      "name": "Sudhir Tibrewal",
+      "avatar_url": "https://avatars1.githubusercontent.com/u/12657704?v=4",
+      "profile": "https://github.com/sudhirtibrewal",
+      "contributions": [
+        "code"
```

Comment:
you can also add  "ideas"m, which is planning

Reply:
No worries, Ill make this change.